weblate_schemas/message.py

```diff
@@ -0,0 +1,106 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: MIT
+
+"""Message class implementation for Weblate Fedora Messaging."""
+
+from datetime import datetime
+
+from weblate_schemas import load_schema
+from fedora_messaging import message
+
+
+class BaseMessage(message.Message):
+    """Inherit the Message class from fedora_messaging."""
+
+    def __init__(self, **kwargs) -> None:
+        """Initialize BaseMessage class."""
+        super().__init__(**kwargs)
+
+    @property
+    def app_name(self) -> str:
+        """Return the app name."""
+        return "Weblate"
+
+    @property
+    def app_icon(self) -> str:
+        """Return the App icon URL."""
+        return "https://weblate.org/static/weblate-128.png"
+
+
+class WeblateMessage(BaseMessage):
+    """Actual Weblate message class which uses the Messaging schema."""
+
+    def __init__(self, **kwargs) -> None:
+        """Initialize the WeblateMessage class with loading of the body_schema."""
+        super().__init__(**kwargs)
+        self.body_schema = load_schema("weblate-messaging.schema.json")
+
```

Comment:
At least `summary` should also be present, based on https://pagure.io/fedora-infrastructure/issue/8291#comment-857387.

Reply:
I would also recommend to add properties that are used by FMN as `usernames` and `packages`. See https://fedora-messaging.readthedocs.io/en/stable/user-guide/messages.html#example-schema

Also each Fedora application should have `app_name` property.